src/package.rs

```diff
@@ -199,16 +199,8 @@ impl Package {
             .status()?;
         if !status.success() {
             // The merge failed! Abort the merge and tell the user to fix it.
```

Comment:
This comment doesn't describe what this actually does anymore, lets just remove it
```suggestion
```